modules/op-release-notes-1-20-0.adoc

```diff
@@ -0,0 +1,289 @@
+
+.{tekton-chains}
+* Pod anti-affinity rules are not applied to `tekton-chains-controller` replicas.
+
+.{tekton-hub}
+* The `git-clone` task downloaded from {tekton-hub} displays version `0.9` instead of `0.10`. This occurs because the system sorts version strings lexicographically rather than using semantic versioning.
+
+[id="pipelines-fixed-issues-1-20-0"]
+=== Fixed issues
+
+.Pipelines
+* Before this update, the {pipelines-shortname} controller did not apply the `managed-by: tekton-pipelines` label when deploying `PipelineRuns` and `TaskRuns` with Helm. In addition, pods created by the controller could have the `app.kubernetes.io/managed-by` label overridden by values set by a `TaskRun` or `PipelineRun`. With this update, the controller consistently applies the correct labels, and pods use the default `app.kubernetes.io/managed-by` value.
+
+* Before this update, the {OCP} Console stated a 'Cancelling' state for `PipelineRun` objects with failed `finally` tasks, even when the actual status of the `PipelineRun` was `Canceled`, causing confusion. With this update, the issue is fixed.
+
+* Before this update, the `PipelineRun` Events tab did not show events, due to an issue with the Pipeline Console plugin. As a consequence, you could not monitor the events in real-time. With this update, the issue is fixed.
+
+* Before this update, the pipeline builder fetched only a limited number of tasks from {artifact-hub}, resulting in some tasks not being available. This resulted in an incomplete pipeline creation. With this update, the interface fetches the complete list of tasks from {artifact-hub} on search, showing all available tasks in the pipeline builder.
+
+* Before this update, running `fbc-fips-check-oci-ta` tasks during git cloning caused temporary resource unavailability, stalling the git resolver and preventing the resolution of `ResolutionRequests`. This resulted in a build failure with the following error message: `cannot fork() for remote-https: Resource temporarily unavailable`. With this update, the issue is fixed.
+
+* Before this update, `TaskRun` and `CustomRun` creation could fail immediately on transient mutating-webhook timeouts, causing flakiness on busy clusters. With this update, the system  applies exponential backoff, configurable via the `wait-exponential-backoff`  config map and controlled by the `enable-wait-exponential-backoff` setting:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: feature-flags
+  namespace: tekton-pipelines
+data:
+  enable-wait-exponential-backoff: "true"
+#...
+----
+
+* Before this update, the controllers used a fixed thread count, limiting concurrency. With this update, you can override `threads-per-controller` by setting the `THREADS_PER_CONTROLLER` environment variable, allowing finer control over controller concurrency.
```

Comment:
Is it on a specific controller ? I thought tektoncd/pipeline supported this for a long time already.